index.html

```diff
@@ -715,8 +719,8 @@ <h2>
     const observer = new ComputePressureObserver(pressureChange);
     observer.observe("cpu");
 
-    // observe.disconnect() will be called promptly, no time to wait for
-    // callback. Latest entries are needed immediately or will be lost otherwise.
+    // observe.disconnect() will be called promptly, no need to wait for
+    // callback. Latest entries must be fetched or will be lost otherwise.
```

Comment:
no need to wait for the callback to be invoked during next event loop rotation. Pending state change records delivery must be read before, otherwise they will be cleared.